.env

```diff
@@ -0,0 +1,48 @@
+# Environment for local demo network.
+# This file is meant to work with docker-compose.yaml
+
+RUST_LOG=info,libp2p=off
+RUST_LOG_FORMAT=full
+
+L1_BLOCK_TIME_SEC=3
+
+# Internal port inside container
+ESPRESSO_WEB_SERVER_PORT=40000 
+ESPRESSO_ORCHESTRATOR_PORT=40001
+ESPRESSO_CONSENSUS_SERVER_PORT=40002
+ESPRESSO_DA_SERVER_PORT=40003
+ESPRESSO_SEQUENCER_DA_SERVER_URL=http://da-server:$ESPRESSO_WEB_SERVER_PORT
+ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL=http://consensus-server:$ESPRESSO_WEB_SERVER_PORT
+ESPRESSO_SEQUENCER_ORCHESTRATOR_URL=http://orchestrator:$ESPRESSO_ORCHESTRATOR_PORT
+ESPRESSO_SEQUENCER_API_PORT=50000
+ESPRESSO_SEQUENCER1_API_PORT=50001
+ESPRESSO_SEQUENCER2_API_PORT=50002
+ESPRESSO_SEQUENCER3_API_PORT=50003
+ESPRESSO_SEQUENCER4_API_PORT=50004
+ESPRESSO_SEQUENCER_URL=http://sequencer0:$ESPRESSO_SEQUENCER_API_PORT
+ESPRESSO_SEQUENCER_STORAGE_PATH=/store/sequencer
+ESPRESSO_SEQUENCER_L1_PORT=8545
+ESPRESSO_SEQUENCER_L1_WS_PORT=8546
+ESPRESSO_SEQUENCER_L1_PROVIDER=http://demo-l1-network:$ESPRESSO_SEQUENCER_L1_PORT
+ESPRESSO_SEQUENCER_L1_WS_PROVIDER=ws://demo-l1-network:$ESPRESSO_SEQUENCER_L1_WS_PORT
+ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG=true
+ESPRESSO_SEQUENCER_HOTSHOT_ADDRESS=0x0116686e2291dbd5e317f47fadbfb43b599786ef
+ESPRESSO_SEQUENCER_ETH_MNEMONIC="test test test test test test test test test test test junk"
+ESPRESSO_SEQUENCER_HOTSHOT_ACCOUNT_INDEX=5
+ESPRESSO_COMMITMENT_TASK_PORT=60000
+
+# Example rollup demo variables
+ESPRESSO_DEMO_ROLLUP_PORT=8084
+ESPRESSO_DEMO_L1_HTTP_PROVIDER=$ESPRESSO_SEQUENCER_L1_PROVIDER
```

Comment:
@ImJeremyHe let's remove the unused ones and put the values of the 3 ones that are used for the example rollup here.